controllers/topics.controllers.js

```diff
@@ -75,3 +77,23 @@ exports.getCommentsByArticleId = (req, res, next) => {
       next(err);
     });
 };
+
+exports.deleteCommentById = (req, res, next) => {
+  const { comment_id } = req.params;
+  checkCommentExists(comment_id)
+    .then(() => {
+      return removeCommentById(comment_id);
+    })
+    .then((rows) => {
+      if (rows.length === 1) {
+        res.status(204).send({ message: "Error Check" });
```

Comment:
I would remove this response object, as 204's can't send any response body anyway (as you've found in your test)